playwright/e2e/crypto/dehydration.spec.ts

```diff
@@ -38,7 +38,7 @@ test.describe("Dehydration", () => {
         // Reset the identity key
         const settings = await app.settings.openUserSettings("Encryption");
         await settings.getByRole("button", { name: "Verify this device" }).click();
-        await page.getByRole("button", { name: "Proceed with reset" }).click();
+        await page.getByRole("button", { name: "Can't confirm?" }).click();
```

Comment:
just for future reference: it's much nicer to review changes where the author has at least aimed for the tests to pass after each commit (so the tests and app are updated in sync)